sphinx-docs/Server-Configuration.md

```diff
@@ -109,3 +110,46 @@ Be sure to change these settings to match your specific LDAP environment.
 
 Note that adding the `ldap` section will disable any accounts listed in the `users` section of the config file;
 only LDAP will be used for logging in.
+
+## Setting Custom Login Handlers
+By default, users authenticate to CALDERA by providing credentials (username and password) in the main login page.
+These credentials are verified using CALDERA's internal user mapping, or via LDAP if LDAP login is enabled for CALDERA.
+If users want to use a different login handler, such as one that handles SAML authentication or a login handler provided
+by a CALDERA plugin, the `auth.login.handler.module` keyword can be added to the CALDERA configuration file.
+The value must be either `default` (to use the default login handler) or a Python import path string corresponding
```

Comment:
Since `auth.login.handler.module` is already in the default conf file should this line change a little bit to something along the lines of:

```
If users want to use a different login handler, such as one that handles SAML authentication or a login handler provided by a CALDERA plugin, the `auth.login.handler.module` keyword in the CALDERA configuration file must be changed from its value of 'default' (used to load the default login handler).
The configuration value, if not `default`, must be a Python import path corresponding to the custom login handler relative to the main CALDERA directory (e.g. `auth.login.handler.module: plugins.app.my_custom_handler`).
```

Reply:
I actually just realized the line isn't in the default config file (it's also set to use the default handler if nothing is specified in the config). I'll add in a small PR to add that in the default config file and then will reword this section as you suggested

Reply:
Oh, I'm blind. It actually is in the default config file. Apologies! Will reword now